brands/amenity/cinema.json

```
@@ -80,6 +80,17 @@
       "short_name": "Century"
     }
   },
+  "amenity/cinema|CGR": {
+    "countryCodes": ["fr"],
+    "matchNames": ["cgr"],
+    "tags": {
+      "amenity": "cinema",
+      "brand": "CGR",
+      "brand:wikidata": "Q2931406",
+      "brand:wikipedia": "fr:Circuit_Georges-Raymond",
```

Wiki tags need to have underscore `_` replaced with space ` `..
```suggestion
      "brand:wikipedia": "fr:Circuit Georges-Raymond",
```

---

Ok, i didn't know that rule. I copied the brand wikipedia from URL. I will correct this.